source/text/implementation/vss-implementation-string_handlers.ads

```diff
@@ -300,4 +308,11 @@ package VSS.Implementation.String_Handlers is
    --  terminators. Line terminator (character or combination of characters)
    --  are removed unless Keep_Terminator is set to True.
 
+   not overriding procedure Get_Case_Mapping
```

Comment:
why Get_Case_Mapping is not a function?

Reply:
Passing of just address as parameter is faster than return of object.